src/UI/src/components/Application/EditApplicationView.vue

```diff
@@ -383,11 +383,14 @@ export default {
       });
     },
     async onAddEnvironment(environmentId) {
-      const application = await this.addEnvironmentToApplication({
+      await this.addEnvironmentToApplication({
         id: this.id,
         environments: [environmentId],
       });
-      this.setApplication(application);
+
+      this.$router.replace({
+        name: "Application_Created",
```

Comment:
was genau macht das?

Reply:
navigate to the page where the new created client is shown